csrc/ops/composite.cpp

```diff
@@ -106,6 +107,25 @@ TensorView* _matmul_tt(TensorView* a, TensorView* b) {
   return tv2;
 }
 
+TensorView* matmul(TensorView* a, TensorView* b) {
+  NVF_CHECK(a->nDims() == b->nDims());
+  NVF_CHECK(a->nDims() == 2 || a->nDims() == 3);
```

Comment:
Are we supporting `a->nDims() == 3` in this API version? If not, we should remove that condition